guides/source/caching_with_rails.md

```diff
@@ -3,12 +3,25 @@
 Caching with Rails: An Overview
 ===============================
 
-This guide is an introduction to speeding up your Rails app with caching.
+This guide is an introduction to speeding up your Rails application with caching.
+
+Caching means to store the content generated during the request-response cycle
+and to reuse it when responding to similar requests.
+
+Caching is arguably one of the hardest problems in computer science. At the same
```

Comment:
`computer science` sounds too abstract for me. That's hardly what most web developers do in day to day development.

I think it would be better to just remove the first sentence and go with:

```
Caching is often the most effective way to boost an application's performance.
```